roles/edpm_nova/templates/nova_compute.json.j2

```diff
@@ -27,6 +27,7 @@
         "/etc/multipath:/etc/multipath:z",
         "/etc/multipath.conf:/etc/multipath.conf:ro",
         "/etc/iscsi:/etc/iscsi:ro",
+        "/etc/nvme:/etc/nvme:z",
```

Comment:
+1 to uncodiotnally adding this.

z will force /etc/nvme to be relabled as contaienr_t

so if anything on the host is going to use this that is probably incorrect

Reply:
Ack, will change this in the next update.